app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -1,21 +1,16 @@
+<% campaign = CampaignDecorator.new(Campaign.current) %>
```

Comment:
Used for simplifying the handling of the sidebar image and the 3 uses of the main tag (the first of the featured tags).